### DISCUSSION TITLE
Deployment with Sql server and prisma

### DISCUSSION THREAD

**Discussion by johnfung95** [vercel/community#2858] [Help] [answered]
### Summary

Hi, I have an application to apply using SQL server as the database and Prisma to connect. However, I am unsure how to deploy the project on Vercel. Any advice?

### Steps to Reproduce

database connection string: sqlserver://xxx.....

How can I also initiate the Prisma Client though the project deployment process?

**Comment by amyegan** [comment, marked as answer]
Hi @johnfung95. I don't know the specifics of your project, so I can't tell exactly what is going wrong. But I have a few resources that I think will help.
- Prisma's guide on how to deploy to Vercel: https://www.prisma.io/docs/guides/deployment/deployment-guides/deploying-to-vercel
- Vercel's guide on building a fullstack app with Next.js, Prisma, and Vercel Postgres: https://vercel.com/guides/nextjs-prisma-postgres
- Project templates that show how Prisma can be used in a Vercel project: https://vercel.com/templates?search=prisma
